src/pages/_error.js

```diff
@@ -28,15 +27,9 @@ class Error extends React.Component {
     this.state = {};
```

Comment:
If the state is not used anymore we can remove it (as well as the `constructor`)

Reply:
Done.